### PR TITLE
feat(teams): Remove role and add ability to remove members from team member list

### PR DESCRIPTION
**Pull request by macqueen**
depends on https://github.com/getsentry/sentry/pull/7236

### CODE REVIEW

File: src/sentry/static/sentry/app/__mocks__/api.jsx [hunk 1]
```diff
@@ -55,6 +55,15 @@ class Client {
     });
   }
 
+  uniqueId() {
+    let s4 = () => {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/__mocks__/api.jsx]: 🔪

File: src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx [hunk 2]
```diff
@@ -42,6 +45,37 @@ const TeamMembers = createReactClass({
...
+      this.api,
+      {
+        orgId: params.orgId,
+        teamId: params.teamId,
+        memberId: member.id,
+      },
+      {
+        success: () => {
+          this.setState({
+            memberList: this.state.memberList.filter(m => {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx]: does this api request return something you can use to do the filtering? not a big deal

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Review by MaxBittker — APPROVED** [review]
code looks good to me, you should post screenshots maybe though for chrissy/sara? or just merge

File: src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx (hunk 2 above)
Comment (macqueen) [reply src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx]: it returns the team, so nothing really to help with filtering

**Comment by macqueen** [comment]
here's a screenshot:
screenshot 2018-02-13 17 30 35

chrissy is going to do a bunch of design stuff to this and other teams pages after this is merged though, so i'm not too worried about making this look nice.

**Comment by MaxBittker** [comment]
/Users/maxbittker/workspace/sentry/src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

**Pull request merged**